pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -73,6 +75,20 @@ const (
 	clusterImageSetFoundReason            = "ClusterImageSetFound"
 )
 
+var (
+	metricClusterDeploymentInstallRetriesTotal = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "hive_cluster_deployment_install_retries_total",
```

Comment:
Correct me if I'm wrong but I believe we should drop the _total suffix now that it's a gauge.